### PR TITLE
Update grafana-performance-overview-dashboard.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-performance-overview-dashboard.md [hunk 1]
```diff
@@ -6,11 +6,23 @@ aliases: ['/zh/tidb/v6.0/grafana-performance-overview-dashboard']
 
 # Performance Overview 面板重要监控指标详解
 
-使用 TiUP 部署 TiDB 集群时，你可以一键部署监控系统 (Prometheus & Grafana)。监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+## 查看方式
```
Comment (shichun-0415) [inline grafana-performance-overview-dashboard.md:9]: 一级标题和二级标题之间、二级标题和三级标题之间缺少衔接，另外，前面的内容结构建议优化，加一点内容介绍 Performance Overview 这个面板。这么改可以吗：
> # Performance Overview 面板重要监控指标详解
>
> Performance Overview Dashboard 是 TiDB v6.1.0 引入的性能分析和优化工具，借助该工具，你可以高效地进行性能分析，确认用户响应时间的瓶颈是否在数据库中。如果数据库是整个系统的瓶颈，你可以通过数据库时间概览和 SQL 延迟的分解，定位数据库内部的瓶颈点，并进行针对性的优化。

> 本文介绍如何查看 Performance Overview Dashboard，并解读面板重要监控指标。
> 
> # 通过 TiDB Dashboard Metrics 查看
> xxx
> # 通过 Grafana 查看
> xxx

File: grafana-performance-overview-dashboard.md [hunk 2]
```diff
@@ -6,11 +6,23 @@ aliases: ['/zh/tidb/v6.0/grafana-performance-overview-dashboard']
...
+
+### Grafana
+
+使用 TiUP 部署 TiDB 集群时，你可以一键部署监控系统 (Prometheus & Grafana)，并在 Grafana 中查看。监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+
+![performance overview](/media/performance/grafana_performance_overview.png)
+
+## 面板监控指标详解
+
+按总分结构对 TiDB、TiKV、PD 的性能指标进行编排组织，包含了以下三部分内容：
```
Comment (shichun-0415) [inline grafana-performance-overview-dashboard.md:25]:
```suggestion
Performance Overview Dashboard 按总分结构对 TiDB、TiKV 和 PD 的性能指标进行了编排组织，包含以下三部分内容：
```

File: grafana-performance-overview-dashboard.md (hunk 1 above)
Comment (lilyjazz) [reply grafana-performance-overview-dashboard.md:9]: 我觉得可以。

File: grafana-performance-overview-dashboard.md [hunk 3]
```diff
@@ -6,53 +6,67 @@ aliases: ['/zh/tidb/v6.0/grafana-performance-overview-dashboard']
...
+## 面板监控指标详解
+
+Performance Overview 面板按总分结构对 TiDB、TiKV 和 PD 的性能指标进行了编排组织，包含以下三部分内容：
 
 - 总的概览：数据库时间和 SQL 执行时间概览。通过颜色优化法，你可以快速识别数据库负载特征和性能瓶颈。
 - 资源负载：关键指标和资源利用率，包含数据库 QPS、应用和数据库的连接信息和请求命令类型、数据库内部 TSO 和 KV 请求 OPS、TiDB 和 TiKV 的资源使用概况。
 - 自上而下的延迟分解：Query 延迟和连接空闲时间对比、Query 延迟分解、execute 阶段 TSO 请求和 KV 请求的延迟、TiKV 内部写延迟的分解等。
 
-借助 Performance Overview Dashboard，你可以高效地进行性能分析，确认用户响应时间的瓶颈是否在数据库中。如果数据库是整个系统的瓶颈，通过数据库时间概览和 SQL 延迟的分解，定位数据库内部的瓶颈点，并进行针对性的优化。详情请参考 [TiDB 性能分析和优化方法](/performance-tuning-methods.md)。
+借助 Performance Overview 面板，你可以高效地进行性能分析，确认用户响应时间的瓶颈是否在数据库中。如果数据库是整个系统的瓶颈，通过数据库时间概览和 SQL 延迟的分解，定位数据库内部的瓶颈点，并进行针对性的优化。详情请参考 [TiDB 性能分析和优化方法](/performance-tuning-methods.md)。
```
Comment (YiniXu9506) [inline grafana-performance-overview-dashboard.md:33]: Is this paragraph duplicated with the one on line 9?

File: grafana-performance-overview-dashboard.md [hunk 4]
```diff
@@ -6,53 +6,65 @@ aliases: ['/zh/tidb/v6.0/grafana-performance-overview-dashboard']
...
-使用 TiUP 部署 TiDB 集群时，你可以一键部署监控系统 (Prometheus & Grafana)。监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+Performance Overview 面板是 TiDB v6.1.0 引入的性能分析和优化工具，借助该工具，你可以高效地进行性能分析，确认用户响应时间的瓶颈是否在数据库中。如果数据库是整个系统的瓶颈，你可以通过数据库时间概览和 SQL 延迟的分解，定位数据库内部的瓶颈点，并进行针对性的优化。详情请参考 [TiDB 性能分析和优化方法](/performance-tuning-methods.md)。
 
-目前 Grafana Dashboard 整体分为 PD、TiDB、TiKV、Node\_exporter、Overview、Performance\_overview 等。
+本文介绍如何查看 Performance Overview 面板，并解读面板重要监控指标。
 
-Performance Overview Dashboard 按总分结构对 TiDB、TiKV、PD 的性能指标进行编排组织，包含了以下三部分内容：
+## 通过 TiDB Dashboard Monitoring 查看
+
+进入 TiDB Dashboard 中，进入指标页面查看。
```
Comment (shichun-0415) [inline grafana-performance-overview-dashboard.md:15]: 看图片，进入的不是指标页面，而是监控 (Monitoring) 页面吧？
```suggestion
启动 TiDB Dashboard，左侧面板选择监控 (Monitoring)，即可看到 Performance Overview 页面。
```

**Review by YiniXu9506 — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: grafana-performance-overview-dashboard.md [hunk 5]
```diff
@@ -6,53 +6,65 @@ aliases: ['/zh/tidb/v6.0/grafana-performance-overview-dashboard']
...
-Performance Overview Dashboard 按总分结构对 TiDB、TiKV、PD 的性能指标进行编排组织，包含了以下三部分内容：
+## 通过 TiDB Dashboard Monitoring 查看
+
+启动 TiDB Dashboard，左侧面板选择监控 (Monitoring)，即可看到 Performance Overview 页面。
+
+![指标页面](/media/dashboard/dashboard-monitoring.png)
+
+## 通过 Grafana 查看
+
+使用 TiUP 部署 TiDB 集群时，你可以一键部署监控系统 (Prometheus 和 Grafana)，并在 Grafana 中查看 Performance Overview 面板。监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
```
Comment (Oreoxmt) [inline grafana-performance-overview-dashboard.md:21]:
```suggestion
使用 TiUP 部署 TiDB 集群时，你可以一键部署监控系统（Prometheus 和 Grafana），并在 Grafana 中查看 Performance Overview 面板。监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
```

是否需要说明如何在 Grafana 中查看 Performance Overview 面板？

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**